Security considerations: Point into corr-clar-future (#31)
core-wg/draft-dns-over-coap: draft-ietf-core-dns-over-coap.md

  I-D.lenders-core-dnr: core-dnr
  I-D.amsuess-core-cachable-oscore: cachable-oscore
  DoC-paper: DOI.10.1145/3609423
  amp-0rtt:

>    title: PR #40 "Amplification and 0-RTT" on "CoAP: Corrections and Clarifications"

Reviewer: ```suggestion
   title: 'PR #40 "Amplification and 0-RTT" on "CoAP: Corrections and Clarifications"'
```

   date: 2024-09-25
   format:
     HTML: https://github.com/core-wg/corrclar/pull/40

>    note: |
>      It is expected that that PR will be merged way ahead of this document's publication;

Reviewer: ```suggestion
   ann: |
     Note: It is expected that that PR will be merged way ahead of this document's publication;
```

Consequently, it is of little concern to leverage the benefits of CoAP caching by setting the ID to
0.

General CoAP security considerations apply.
Exceeding those in {{Section 11 of RFC7252}},
the request patterns of DoC make it likely that long-lived security contexts are maintained:

> {{amp-0rtt}} goes into more detail on what can and needs to be done

Reviewer: ```suggestion
{{amp-0rtt}} goes into more detail on what needs to be done
```

Unless I am missing something regarding IETF lingo, the "can" is superfluous.

> when those are resumed from a new address.

Reviewer: With the opening "General CoAP security considerations apply." wouldn't it make more sense to make this the first paragraph?

Author: Moved to the top.

> when those are resumed from a new address.

Reviewer: ```suggestion
when those are resumed from a new source address or port.
```

Author: Maybe 'new endpoint' – source-address-and-port is a UDP/TCP thing. Thinking of OSCORE, you're also in a "new endpoint" situation if after requests over TCP from an address/port combination, all of a sudden the requests come from the same UDP address/port. (One might argue that it's very likely that this is return routable if TCP was, the same argument can also be made for same-IP-different-port, and then we'd have to think about NAT, and I don't want to think about NAT).

Author: Updated to "new endpoint" – that fine?
